src/lorawan_rxdelay.erl

```diff
@@ -0,0 +1,232 @@
+%%%-------------------------------------------------------------------
+%% @doc
+%% == LoRaWAN RxDelay ==
+%% See LoRaWAN Link Layer spec v1.0.4
+%% sections:
+%% - 3.3 Receive Windows
+%% - 4.2.1 Frame types (FType bit field), for Join Accept
+%% - 5.7 Setting Delay between TX and RX (RXTimingSetupReq, RXTimingSetupAns)
+%% - 6.2.6 Join-Accept frame, which includes RXDelay
+%% @end
+%%%-------------------------------------------------------------------
+-module(lorawan_rxdelay).
```

Comment:
Change the module name to lora_rxdelay.  This will be consistent with other modules in this library.
It also avoids a name conflict with the lorawan_rxdelay module in router.

Reply:
Fixed in commit 9ca02b5